Assets/World/Global.gd

```diff
@@ -111,7 +111,7 @@ enum ResourceType {
 	# 92-99 reserved for services
 }
 
-enum Faction {
+enum FACTION {
```

Comment:
Enums should be PascalCase.
https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html#constants-and-enums
```suggestion
enum Faction {
```

Reply:
`Faction` conflicts with the `Faction` class name. Maybe we should rename the enum to `Factions`?

Reply:
Since the `Faction` class is a `Resource` it could be called `FactionData`, `FactionRes` or something similar (you can pick what sounds better).

Reply:
I think `FactionData` seems more self explanatory. I'm going with that one.